modules/odc-interacting-with-applications-and-components.adoc

```diff
@@ -18,3 +18,9 @@ This feature is available only when you create applications using the *From Git*
 * Hover your cursor over the lower left icon on the Pod to see the name of the latest build and its status. The status of the application build is indicated as *New* (image:odc_build_new.png[title="New Build"]), *Pending* (image:odc_build_pending.png[title="Pending Build"]), *Running* (image:odc_build_running.png[title="Running Build"]), *Completed* (image:odc_build_completed.png[title="Completed Build"]), *Failed* (image:odc_build_failed.png[title="Failed Build"]), and *Canceled* (image:odc_build_canceled.png[title="Canceled Build"]).
 * Use the *Shortcuts* menu listed on the upper-right of the screen to navigate components in the *Topology* view.
 * Use the *List View* icon to see a list of all your applications and use the *Topology View* icon to switch back to the *Topology* view.
+* Use the *Display* dropdown menu to manipulate the various application groupings. These are automatically available depending on your project structure:
```

Comment:
- Use exact UI terms: Display Options. Always test with the appropriate cluster, as in 4.6 feature in 4.6 etc.
- s/manipulate/filter
- s/dropdown menu/drop-down list

Reply:
As far as possible, use exact sequence of the displayed options.
Also note the Show and the Expand sections in the options. I have altered the text for this accordingly and suggested above.

Reply:
![Screenshot from 2020-07-13 20-29-26](https://user-images.githubusercontent.com/16647625/87319761-b350fa80-c547-11ea-9dc8-7dfd59951023.png)